self-hosted-login/okta-aspnet-mvc-example/Controllers/AccountController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Owin.Security;
+using Microsoft.Owin.Security.Cookies;
+using Microsoft.Owin.Security.OpenIdConnect;
+using System.Web;
+using System.Web.Mvc;
+
+namespace okta_aspnet_mvc_example.Controllers
+{
+    public class AccountController : Controller
+    {
+        
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Login(FormCollection form)
+        {
+            if (!HttpContext.User.Identity.IsAuthenticated)
+            {
+                var properties = new AuthenticationProperties();
+                properties.Dictionary.Add("sessionToken", form.Get("sessionToken"));
+                properties.RedirectUri = "/Home/About";
+
+                HttpContext.GetOwinContext().Authentication.Challenge(properties,
+                    OpenIdConnectAuthenticationDefaults.AuthenticationType);
```

Comment:
Instead of `OpenIdConnectAuthenticationDefaults`, use `OktaDefaults.MvcAuthenticationType`